projects/demo/src/modules/components/input-files/examples/6/index.html

```diff
@@ -0,0 +1,6 @@
+<tui-input-files
+    *ngIf="!control.value"
+    accept="image/*"
+    capture="user"
+    [formControl]="control"
+></tui-input-files>
```

Comment:
Let's change text so it reflects camera capture. Also when you select value it just disappears because of `ngIf` which is very unclear. Let's instead show the image user have taken and allow them to remove/redo it.

Reply:
fixed